### PR TITLE
Bunch-o' Wiring

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## Notes:
This is in support of the following PRs:
- Atomic: https://github.com/yalesites-org/atomic/pull/1
- Component Library: https://github.com/yalesites-org/component-library-twig/pull/68

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,7 +43,7 @@
         "drupal/redirect": "^1.7",
         "drupal/twig_tweak": "^3.1",
         "drupal/xmlsitemap": "^1.2",
-        "yalesites-org/atomic" : "dev-main"
+        "yalesites-org/atomic": "dev-YALB-253-theming-support-artifacts"
```
Comment (nJim) [inline composer.json:46]: We will change this once merging the atomic pr.

File: config/sync/block.block.user_account_menu.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+status: true
+dependencies:
+  config:
+    - system.menu.account
+  module:
+    - system
+  theme:
+    - atomic
+id: user_account_menu
+theme: atomic
```
Comment (nJim) [inline config/sync/block.block.user_account_menu.yml:12]: Ad discussed in the related PR for the atomic theme, I think this block can be deleted and replaced with a utility menu block.

File: config/sync/block.block.site_branding.yml [hunk 3]
```diff
@@ -1,16 +1,14 @@
-uuid: 231a32e5-a3a4-471c-9e9b-5d9c0f85a99b
+uuid: a78f6bdc-4ce5-4aa1-aca7-e495c46d8b6f
```
Comment (nJim) [inline config/sync/block.block.site_branding.yml:1]: Works for me. I'm just curious of the pros and cons with using a site branding block vs just setting the image in the template directly.

**Review by nJim — APPROVED** [review]

**Comment by nJim** [comment]
Approved in its current form, but let's hold off on merging until the atomic theme work is complete. At this time we change the theme branch and may need to change the name of the utility menu.

File: config/sync/block.block.site_branding.yml (hunk 3 above)
Comment (ModulesUnraveled) [reply config/sync/block.block.site_branding.yml:1]: This is actually just a placeholder for now. It's not doing anything. Without it though, the footer wouldn't show on the site, so I just needed something in the region in order for Drupal to render the footer. We will remove it eventually.

File: config/sync/block.block.user_account_menu.yml (hunk 2 above)
Comment (ModulesUnraveled) [reply config/sync/block.block.user_account_menu.yml:12]: Done.

**Pull request merged**